packages/verification/src/assets/styles/components/VerificationItem.scss

```diff
@@ -61,6 +61,24 @@
     padding: 0 0 $spacer * .5 0;
   }
 
+  .issuer-verifiedby {
+    position: relative;
+  }
+
+  .issuer-image {
+    position: absolute;
+    top: 0;
+    width: 64px;
+    height: 64px;
+    display: inline-block;
+    margin-left: 10px;
```

Comment:
Is it necessary to position it absolute? You have now a mix between absolut (top: 0) and margin-left. As long as it works, it's good enough for now I guess.

Reply:
It's the best solution i came up following the design, maybe there are better options..?

Reply:
There is probably a cleaner solution, but nothing I can write without trying by myself. The new verification tool has a slightly different structure, so this needs to be taken over to the new anyway, so it doesn't matter as long as it works ;-)